src/defs.h

```diff
@@ -563,12 +564,16 @@ extern enum stack_trace_modes stack_trace_mode;
 #  define stack_trace_mode STACK_TRACE_OFF
 # endif
 extern unsigned max_strlen;
+extern bool truncate_output;
+extern struct sysinfo info;
```

Comment:
`struct sysinfo` is an implementation detail, please do not turn it into an interface.